scm-ui/ui-types/src/Config.ts

```diff
@@ -33,7 +35,7 @@ export type Config = {
   realmDescription: string;
   disableGroupingGrid: boolean;
   dateFormat: string;
-  anonymousAccessEnabled: boolean;
```

Comment:
I'm not sure if it is safe to remove the `anonymousAccessEnabled` signature. What about plugins which are relying on this property.